<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="Filters">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>224</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Brightening</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Darkening</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Emboss</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Find Edges</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gaussian Filter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gaussian Smoothing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Invert</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Median Filter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Motion Blur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Only Blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Only Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Only Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sharpen</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>192</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose your image filter...</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="fromFile">
       <property name="text">
        <string>Choose your image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toFile">
       <property name="text">
        <string>Choose where to save your image</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>186</width>
      <height>28</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="Cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Done">
     <property name="text">
      <string>Let's do it!</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="formatImage"/>
    <addaction name="getHelp"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="formatImage">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Available formats</string>
   </property>
  </action>
  <action name="getHelp">
   <property name="text">
    <string>Get Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
